README.md

```diff
@@ -57,6 +57,9 @@ The application makes calls to
 [case orchestration service](https://github.com/hmcts/nfdiv-case-orchestration-service).
 Make sure you have this running to run the frontend e2e.
 
+Some older versions of MacOS would require zsh to be installed and setup. This could help to avoid seeing
```

Comment:
I would say instead of this could you please update something like use zsh while executing scripts in bin folder.

Reply:
this has been addressed now.